src/Razor/src/Microsoft.NET.Sdk.Razor/build/netstandard2.0/Sdk.Razor.CurrentVersion.props

```diff
@@ -74,4 +74,8 @@ Copyright (c) .NET Foundation. All rights reserved.
     <None Remove="**\*.razor" />
   </ItemGroup>
 
+  <Import 
+    Project="$(MSBuildThisFileDirectory)Sdk.Razor.StaticAssets.ProjectSystem.props"
```

Comment:
Project="$(MSBuildThisFileDirectory)Sdk.Razor.StaticAssets.ProjectSystem.props"

Reply:
These files should go to the SDK folder so that if we want to import, we can import this like this:

`  <Import Sdk="Microsoft.NET.Sdk.Razor" Project="Sdk.Razor.StaticAssets.ProjectSystem.props" />`

Reply:
Move the files to the SDK folder